docroot/themes/humsci/humsci_basic/templates/menus/macros/nav-menu.twig

```diff
@@ -60,13 +59,13 @@
           nested navigation and the anchor tag will just act as a link.
         #}
         {{ link(item.title, item.url, link_attributes) }}
-
+        
```

Comment:
Could you please delete this extra space that was added?

Reply:
All done!